packages/video-intelligence/src/index.js

```diff
@@ -13,12 +13,6 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
-/*!
```

Comment:
Need!

Reply:
@stephenplusplus Could you add a bug in googleapis/toolkit explaining what this is?